implants/lib/eldritchv2/stdlib/eldritch-libsys/src/std/exec_impl.rs

```diff
@@ -49,10 +51,28 @@ fn handle_exec(
     args: Vec<String>,
     env_vars: HashMap<String, String>,
     disown: bool,
+    input: Option<String>, // Added input option
 ) -> Result<CommandOutput> {
+    // Setup stdin configuration, null if no input is given
+    let stdinpipe = if input.is_some() { Stdio::piped() } else { Stdio::null() };
```

Comment:
Does this work with disown?
Might be an edge case where it holds the disowned process open indefinitely.